Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "cargo-gccrs"
 version = "0.0.1"
-authors = ["Arthur Cohen <arthur.cohen@epita.fr"]
+authors = ["Arthur Cohen <cohenarthur.dev@gmail.com"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+getopts = "0.2"
```

Comment:
Any reason why you use `getopts` over `clap` or `struct-ops`? I don't have a reason why you should use one over the other, but I would make this decision now to not have to rework this later.

Reply:
Rustc uses `getopts` as base for argument parsing. I don't think it is possible to exactly match rustc's argument parsing with `clap` or `struct-opts`.